lib/validators/movies/list.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  name: Joi.string().min(1).max(255).optional(),
```

Comment:
By default all keys are optional so there's no need to add it for every single key. Personally I think it adds noise, but it's completely up to you.